forge-game/src/main/java/forge/game/staticability/StaticAbilityDisableTriggers.java

```diff
@@ -0,0 +1,125 @@
+package forge.game.staticability;
+
+import com.google.common.base.Predicates;
+import com.google.common.collect.Table.Cell;
+
+import forge.game.Game;
+import forge.game.card.*;
+import forge.game.ability.AbilityKey;
+import forge.game.trigger.Trigger;
+import forge.game.trigger.TriggerType;
+import forge.game.zone.ZoneType;
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.util.Map;
+
+public class StaticAbilityDisableTriggers {
+
+    static String MODE = "DisableTriggers";
+
+    public static boolean disabled(final Game game, final TriggerType triggerType, final Trigger regtrig, final Map<AbilityKey, Object> runParams)  {
```

Comment:
one thing i don't know yet is if we need both `final TriggerType triggerType, final Trigger regtrig` when there is already checks if regtrig is the right mode?

i mean i need the regtrig param later for my Read Ahead stuff, so we could skip the triggerType?

but we also could make regtrig nullable and only check the type of triggerType below?
with that we could make a more generic test for the AI

Reply:
Imo let's get rid of the Type

In case of Elesh Norn it can't be decided without looking at the regtrig anyway

Reply:
i would have allowed both params, but i can think about that for later